network.go

```diff
@@ -423,6 +424,60 @@ func (d *Driver) NetworkRemovePrefix(nur *NetworkUpdateRequest) (*NetworkDetailR
 	return response, nil
 }
 
+// NetworkAddDestinationprefix adds a prefix to a network
+func (d *Driver) NetworkAddDestinationprefix(nur *NetworkUpdateRequest) (*NetworkDetailResponse, error) {
```

Comment:
These mapper functions are deprecated and not supposed to be used anymore. You can leave them here but in metalctl please implement this logic directly.